xchange-core/src/main/java/org/knowm/xchange/dto/Order.java

```diff
@@ -408,12 +421,14 @@ protected Builder(OrderType orderType, CurrencyPair currencyPair) {
       this.currencyPair = currencyPair;
     }
 
+    @JsonProperty("type")
```

Comment:
ok, now I understand why you say that the naming convention in this file is a mess :)
for now the getters/setters of the main class should be the main determinatior of a name of a given property
in a 5.X.X release I think we should ditch the word Order in any property here as it is obvious that it relates to the order